lib/apns/state.ex

```diff
@@ -0,0 +1,16 @@
+defmodule APNS.State do
+  def get(options) do
+    {:ok, queue_pid} = APNS.Queue.start_link
```

Comment:
Better really because you won't have message passing overhead, we don't have to solve the problem of having the queue supervised, etc.